theme/mimemail-message.tpl.php

```diff
@@ -23,7 +23,7 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <?php if ($css): ?>
-    <style type="text/css">
+    <style>
```

Comment:
Hi @alanmels. I'm curious about this change. It's not related to the issue you opened. Is there a reason this needs to be changed? Can you move this to another issue/PR?

The rest looks fine.

Reply:
OK, I see there is a separate issue, but you provided this in a PR that's not linked to that issue.

Can you please add the magic words "Fixes XXX" in the description of the PRs? (You added those words in the commit title, not in the descriptoin). Otherwise the issue will not be closed when this is merged.